streams/src/test/java/org/apache/kafka/streams/tests/StreamsBrokerDownResilienceTest.java

```diff
@@ -45,15 +45,14 @@
     private static final String SINK_TOPIC = "streamsResilienceSink";
 
     public static void main(final String[] args) throws IOException {
-        if (args.length < 2) {
-            System.err.println("StreamsBrokerDownResilienceTest are expecting two parameters: propFile, additionalConfigs; but only see " + args.length + " parameter");
+        if (args.length < 1) {
+            System.err.println("StreamsBrokerDownResilienceTest is expecting one parameter: propFile; but currently sees " + args.length + " parameter(s)");
```

Comment:
I know this wasn't there before, but maybe for this case, we should print out the argument passed.

Reply:
Well. This only triggers is there are zero argument provided. So there is nothing to be printed IMHO. Maybe use:
```
System.err.println("StreamsBrokerDownResilienceTest requires one argument (properties-file) but none provided.");
```